recipes/jwt-cpp/all/conanfile.py

```diff
@@ -8,8 +8,8 @@ class JwtCppConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/Thalhammer/jwt-cpp"
     description = "A C++ JSON Web Token library for encoding/decoding"
-    topics = ("jwt-cpp", "json", "jwt", "header-only", "conan-recipe")
-
+    topics = ("jwt-cpp", "json", "jwt", "header-only", "jose")
```

Comment:
what does mean "jose" ?

Reply:
json object signing and encryption
https://tools.ietf.org/html/rfc7520

Reply:
nice! I didn't know about, thanks!